test/integration/sign_test.go

```diff
@@ -148,3 +149,59 @@ func TestIsImageSigned(t *testing.T) {
 		}
 	}
 }
+
+func TestImagesSigned(t *testing.T) {
```

Comment:
Nit: should we have a case that tests one signed image and one valid, but non-signed image?

Reply:
I'm not sure I understand correctly, "one signed image" and "non-signed" seems to be exclusive to me.

Reply:
I see this is actually handled by `list of valid images`, so ignore the comment. :slightly_smiling_face: